master/static/migrations/20210204000000_add-total-batch.down.sql

```diff
@@ -0,0 +1,32 @@
+-- Replace total_batches with step_id in the checkpoints table.
```

Comment:
Since the release party cluster is up right now, can you time this up and down on it? For context, I've noticed from us to our largest users usually results in a 5-20x increase in duration on migrations like this (relative to the size out latest master/release party).

Reply:
Do you mean time this and put it in the release note?

Reply:
Ah you could put it in the release note, but I just meant time it so we'd know to warn people we help upgrade.